styx-cli/src/main/java/com/spotify/styx/cli/PrettyCliOutput.java

```diff
@@ -117,6 +119,7 @@ private void printBackfillHeader() {
                                      "HALTED",
                                      "ALL TRIGGERED",
                                      "CONCURRENCY",
+                                     "LEFT",
```

Comment:
may be better to move it right after "ALL TRIGGERED". it is naturally to think how many left after seeing not yet all triggered.